### PR TITLE
Adds Ctrl-I As The *twirl Hotkey

### PR DESCRIPTION
**Pull request by Mister-Moriarty**
[Feature] [QoL]

## About the PR:
As is described, the formerly unused Ctrl-I is now used as the *twirl hotkey.

## Why's this needed?
A widely requested hotkey, for all those that want to try to put their firearm away in suave style only to shoot themselves in the foot.
Currently unbinded Ctrl-keys: Ctrl-C, Ctrl-J, Ctrl-K, Ctrl-M, Ctrl-O, Ctrl-U, and Ctrl-V.

## Changelog:
```changelog
(u)Mr. Moriarty
(*)Added Ctrl-I as the *twirl hotkey.
```

### CODE REVIEW

**Comment by TTerc** [comment]
afaik, on tg controls, ctrl+R is the flip hotkey while R is for throwing

**Comment by Mister-Moriarty** [comment]
Hmm, what would make the most sense as a TG *twirl hotkey, or should I leave it unbinded?

File: browserassets/html/admin/hotkeys.html [hunk 1]
```diff
@@ -87,7 +87,7 @@ <h3>Emotes</h3>
 			<li><b>Ctrl+L:</b> Laugh</li>
 			<li><b>Ctrl+N:</b> Nod</li>
 			<li><b>Ctrl+Q:</b> Wave</li>
-			<li><b>Ctrl+R:</b> Flip</li>
```
Comment (ZeWaka) [inline browserassets/html/admin/hotkeys.html:90]: what happened to my poor flip

Comment (Mister-Moriarty) [reply browserassets/html/admin/hotkeys.html:90]: I should pay more attention to the Git Graph...

**Pull request merged**